<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffManagementMainWindow</class>
 <widget class="QMainWindow" name="StaffManagementMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Staff Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutMain">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="styleSheet">
       <string>font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Staff Management</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutContent">
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutEmployeeSection">
        <item>
         <widget class="QGroupBox" name="groupBoxEmployeeList">
          <property name="title">
           <string>List of Staff</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutEmployeeList">
           <item>
            <widget class="QTableWidget" name="tableWidgetEmployee">
             <property name="rowCount">
              <number>6</number>
             </property>
             <property name="columnCount">
              <number>4</number>
             </property>
             <row>
              <property name="text">
               <string>1</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>2</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>3</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>4</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>5</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>6</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Dob</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Role</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxEmployeeDetail">
          <property name="title">
           <string>Staff Detail </string>
          </property>
          <layout class="QGridLayout" name="gridLayoutEmployeeDetail">
           <item row="0" column="2">
            <widget class="QLabel" name="labelEmployeeName">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEditEmployeeName"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditEmployeeID"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelEmployeeID">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEditRole">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditDob"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="labelPassword">
             <property name="text">
              <string>Role</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelUsername">
             <property name="text">
              <string>Dob</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <layout class="QHBoxLayout" name="horizontalLayoutJobTitle"/>
           </item>
           <item row="3" column="0" colspan="4">
            <layout class="QHBoxLayout" name="horizontalLayoutButtons">
             <item>
              <widget class="QPushButton" name="pushButtonSort">
               <property name="text">
                <string>Sort</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSaveStaff">
               <property name="text">
                <string>Save Staff</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonDelete">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonCount_2">
               <property name="text">
                <string>Show All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonCount">
               <property name="text">
                <string>Count </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonTop3">
               <property name="text">
                <string>TOP 3 Employees with oldest age </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
